container-storage-setup.sh

```diff
@@ -1330,6 +1346,7 @@ check_storage_options(){
 # This is used in compatibility mode.
 setup_storage_compat() {
   local current_driver
+  local containerroot;
```

Comment:
Why the trailing `;` here?

Reply:
Was doing coding in C before this and muscle memory kicked in. :-)